src/test/java/org/humancellatlas/ingest/submission/SubmissionEnvelopeServiceTest.java

```diff
@@ -356,10 +356,35 @@ public void testContentLastUpdated() {
         when(file.getUpdateDate()).thenReturn(now);
 
         // when
-        Instant lastUpdateDate = service.getSubmissionContentLastUpdated(submission);
+        Optional<Instant> lastUpdateDate = service.getSubmissionContentLastUpdated(submission);
 
         // then
-        assertThat(lastUpdateDate).isEqualTo(now.toString());
+        assertThat(lastUpdateDate.isPresent()).isTrue();
+        assertThat(lastUpdateDate.get().toString()).isEqualTo(now.toString());
```

Comment:
AssertJ handles Optionals elegantly.
See https://www.baeldung.com/assertJ-java-8-features
```suggestion
        assertThat(lastUpdateDate)
            .isPresent()
            .hasValue(now.toString());
```